### DISCUSSION TITLE
Disable default UI styles

### DISCUSSION THREAD

**Discussion by titantwentyone** [GrapesJS/grapesjs#5688] [Q&A]
Just woindered if there was a way to disable the UI styles provided by GrapesJS (i.e. the way toolbars and panels are rendered). I am looking to control this with Tailwind but find myself having to override a lot of stuff. I saw there is a flag for [`customUI`](https://github.com/GrapesJS/grapesjs/blob/639305d54a7630f6209a4ff151adfe8cdab4d722/src/editor/config/config.ts#L412) but is marked as experimental.

**Comment by artf** [comment]
Yeah, at the moment I'd ignore that configuration. Currently, the better options right now are simple CSS overrides or a complete rewrite of the UI (eg. similar to how https://github.com/GrapesJS/react works)
